xbmc/video/VideoDatabase.cpp

```diff
@@ -11701,17 +11746,25 @@ std::string CVideoDatabase::GetVideoItemTitle(VideoDbContentType itemType, int d
   }
 }
 
-void CVideoDatabase::InitializeVideoVersionTypeTable()
+void CVideoDatabase::InitializeVideoVersionTypeTable(int schemaVersion)
```

Comment:
Unsigned int?

Reply:
> Unsigned int?

Logically yes, but wanted to be consistent at least => `int CVideoDatabase::GetSchemaVersion() const`

Reply:
Right